src/asicverifier/src/main/java/ee/ria/xroad/asicverifier/AsicVerifierMain.java

```diff
@@ -52,7 +53,9 @@ private AsicVerifierMain() {
      * @throws Exception in case of errors
      */
     public static void main(String[] args) throws Exception {
-        if (args.length < 2) {
+        if (args.length == 1 && args[0].equals("version")) {
```

Comment:
A common convention is to use `--version` option for displaying the version of the utility.